migrations/20210927181326_add_refresh_token_parent.up.sql

```diff
@@ -1,8 +1,24 @@
 -- adds parent column
 
 ALTER TABLE auth.refresh_tokens
-ADD COLUMN IF NOT EXISTS parent varchar(255) NULL,
-ADD CONSTRAINT refresh_tokens_token_unique UNIQUE ("token"),
-ADD CONSTRAINT refresh_tokens_parent_fkey FOREIGN KEY (parent) REFERENCES auth.refresh_tokens("token");
+ADD COLUMN IF NOT EXISTS parent varchar(255) NULL;
```

Comment:
Can you please use `text` instead of `varchar(255)`. They're exactly the same in Postgres -- yet don't have the limitation of 255.

Reply:
will keep in mind for future migrations and leave this out in this PR for now - the intent of this change was mainly to make this migration idempotent